ruby_on_rails/app_initialisation.md

```diff
@@ -72,7 +71,22 @@ They are always idempotent (runnable multiple times).
   ```
 
   Also change `system!('bundle install')` to `system!('bundle install --jobs=3 --retry=3')`
-  and uncomment the installation of yarn dependencies with `system('bin/yarn')`
+
+#### If you are using `importmap-rails`
+
+* Add a `bin/dev` file and make it executable. It will be used to start our project.
+
+  ```sh
+  echo "#\!/bin/sh\nset -e\n\nrails s" >> bin/dev && chmod +x bin/dev
+  ```
```

Comment:
To make things uniform between projects using importmaps or node, I'd change the name of the script to `bin/dev`